### ISSUE TITLE
Make tables in reports (QC etc) downloadable

### ISSUE THREAD

**Issue by cookkate** [R-ODAF/R-ODAF_Health_Canada#43]
Would be useful to have a tab-delimited text version of the various tables available. Copying and pasting into excel does seem to work (at least on my machine), but it's clunky and error-prone.

**Comment by mattjmeier** [comment]
So in the DEG_output folder, there should be txt files alongside the Excel files that are produced. Do you mean some other kind of tables? The pathway analysis ones go in their own folder (a subfolder of that one).

Should be named things like ${experiment}DESeq_output_all_genes.txt (instead of xlsx).

Or did you mean something else entirely?

**Comment by cookkate** [comment]
I meant the small tables within the html file, like the one of failed
samples, or the summary of #DEGs per group.

**Comment by mattjmeier** [comment]
Ahhh ok.

The DEG summary is actually output too. And also the render_DESeq_report script also outputs a summary of the summaries as you might recall I wrote some really simple bash code to do that before the holidays.

The QC script also outputs a few text files, which are stupidly dumped into ./data/processed/ ... QC_per_sample.txt, samples_removed.txt, outliers_cluster_analysis.txt. This definitely needs to be cleaned up (i.e., #6 ).

But if there are others you find that you want feel free to add!

**Comment by mattjmeier** [comment]
Oh, also if you're talking about adding like a "download" embedded link into the HTML that would be awesome too.

At one point I had tried embedding excel files... it was horrible because they are huge of course so it made the HTMLs even bigger. Small light tsvs would be fine I'm sure though. Less overhead than the plotly and datatables for sure! Haha

**Comment by cookkate** [comment]
Yeah I'll check the qc dir to see what's there, it probably has something I
can use, but I was thinking more generally--When I was at tbs we added a
little button to every graph that allowed you to download a jpeg file or
see/download a table of the data, and people really liked it.
